File: benchmarks/cpp/fibonacci.cpp
// BENCH_FIBONACCI_N=35
#include <iostream>

static long long fib(int n) {
    if (n <= 1) return n;
    return fib(n - 1) + fib(n - 2);
}

int main() {
    std::cout << fib(35) << '\n';
    return 0;
}

File: benchmarks/python/fibonacci.py
#!/usr/bin/env python3
"""BENCH_FIBONACCI_N=35"""

def fib(n: int) -> int:
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)


def main() -> None:
    print(fib(35))


if __name__ == "__main__":
    main()
